book/operators.md

```diff
@@ -289,3 +289,53 @@ as being omitted:
 > foo "bar" ...[1 2] "not opt" # The null means no argument was given for opt
 [false, bar, null, [1, 2, "not opt"]]
 ```
+
+## Cell path access
```

Comment:
I didn't see any mention of `cell-path literals`. We should probably mention it with an example.

```nu
let cp = $.1.name
[{name: sam rank: 10} {name: bob rank: 7}] | get $cp
```